Readme.md

```diff
@@ -53,23 +53,41 @@ In Jenkins, there are two types of project.
 * Pipeline project: Config through code
 
 ### Create a Freestyle project
-#### Build and Push task
+#### Build task
 
```

Comment:
Add a short description for each task. Build task can help build solution into container images, etc.